environment.yml

```diff
@@ -391,6 +391,7 @@ dependencies:
     - pyflakes==2.1.1
     - pyside2==5.13.2
     - q2-qemistree==2020.1.3
+    - q2-cscs==0.0.2.4
```

Comment:
This entry doesn't make sense unless q2-cscs is conda/pip installable.

Reply:
It can be conda installed: https://anaconda.org/askerdb/q2-cscs

Reply:
Awesome, can you add the channel in the channels section of this file?

Reply:
Added!